azure-pipelines/build.yml

```diff
@@ -60,12 +58,26 @@ extends:
               signType: $(SignType)
             sbom:
               enabled: true
+          outputs:
+          - output: nuget
+            displayName: 'NuGet push to VS feed'
+            condition: and(succeeded(), eq(variables['Build.OfficialRelease'], 'true'))
```

Comment:
Did we even need to push to the VS feed if we push to nuget.org?

Reply:
I was actually going to ask you about that. That was done in the old XamlBehaviorsWpf pipeline, so it was kept.

That said, afaict, XamlBehaviors never did and I don't know the reason for XamlBehaviorsWpf publishing to an internal feed.

Reply:
I think that was an early attempt to publish to nuget.org, not realizing that it only went to an internal feed.